src/components/beginComponent.js

```diff
@@ -5,15 +5,10 @@ const hasRenderBodyKey = Symbol.for("hasRenderBody");
 
 function isInputSerializable(component) {
     var input = component.___input;
-    if (!input) {
-        return true;
-    }
-
-    if (input[hasRenderBodyKey] === true || input.renderBody !== undefined) {
+    if (input && (input[hasRenderBodyKey] === true || input.renderBody !== undefined)) {
```

Comment:
A minifier will completely rewrite this code so I'm thinking it is best to keep the code readable. Here's the output of Google Closure Compiler on the original code (formatted for readability):

```js
function isInputSerializable(a) {
    a = a.___input;
    return !a || !0 !== a[hasRenderBodyKey] && void 0 === a.renderBody ? !0 : !1
};
```